docs/use_cases/met_tool_wrapper/GridStat/GridStat_python_embedding.py

```diff
@@ -0,0 +1,129 @@
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# Compare dummy forecast data to dummy observations. Generate statistics of
+# the results.
+
+##############################################################################
+# Datasets
+# --------
+#
+# | **Forecast:** Dummy text files found in the MET shared directory
+# | **Observation:** Dummy text files found in the MET shared directory
+#
+# | **Location:** All of the input data required for this use case can be found in the met_test sample data tarball. Click here to the METplus releases page and download sample data for the appropriate release: https://github.com/dtcenter/METplus/releases
+# | This tarball should be unpacked into the directory that you will set the value of INPUT_BASE. See 'Running METplus' section for more information.
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case utilizes the METplus GridStat wrapper to search for
+# files that are valid at a given run time and generate a command to run
+# the MET tool grid_stat if all required files are found.
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# GridStat the only tool called in this example. It processes a single run time with three forecast leads. The input data are simple text files with no timing information, so the list of forecast leads simply duplicates the same file multiple times to demonstrate how data is read in via Python Embedding.
+#
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/met_tool_wrapper/GridStat/GridStat_python_embedding.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/met_tool_wrapper/GridStat/GridStat_python_embedding.conf
+
+##############################################################################
+# MET Configuration
+# ---------------------
+#
+# METplus sets environment variables based on the values in the METplus configuration file.
+# These variables are referenced in the MET configuration file. **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!** If there is a setting in the MET configuration file that is not controlled by an environment variable, you can add additional environment variables to be set only within the METplus environment using the [user_env_vars] section of the METplus configuration files. See the 'User Defined Config' section on the 'System Configuration' page of the METplus User's Guide for more information.
```

Comment:
For this sentence "See the 'User Defined Config' section on the 'System Configuration' page of the METplus User's Guide for more information." consider linking the user to this section directly.  The instructions for doing this are listed under the "Linking to another section" of the [METplus Sphinx Documentation Guide](https://docs.google.com/document/d/1A-r-rE2SHB-jl29PblyuaetAG7Rqa2TmPxvFaWNCguY/edit#heading=h.lqgn762oe1tw) in our Google Team Drive.

Reply:
@jprestop , the text you are referring to is found in every use case documentation file. There are a handful of things that should be cleaned up and improved in these across the board. Another example is the way we word the "Running METplus" section requires us to modify the name of the new config 4 times. We also have discussed improving how we show how METplus config variables correspond to MET config variables that we have dropped discussion without fully resolving. We should create a new GitHub issue to review these doc files and come up with a better standard that is more easily maintainable for adding new cases and make sure all of these things are addressed.

Reply:
@georgemccabe Ah!  Thanks for letting me know. I'm not sure I fully grasp the other issues you describe, but I didn't put much time into looking into them right now, either.  :)  Could you please create the GitHub issue, listing the necessary items to clean up and assign at least, you, me, and Lisa?